apps/synapse-portal-framework/src/components/elportal/ELContributeYourData.tsx

```diff
@@ -34,47 +35,25 @@ function ELContributeYourData() {
           padding: { xs: '100px 40px', md: '100px 0' },
         }}
       >
-        <Typography
-          variant="headline1"
-          style={{ color: 'white' }}
+        <PortalSectionHeader
+          centered
+          reverseButtonAndText
+          title="Contribute Your Data"
+          summaryText="If you are a funded portal contributor and ready to upload data to the
+        ELITE Portal, you can begin the data submission process by contacting
+        our data curation team through our service desk."
+          buttonText="Start Contributing"
+          link="https://sagebionetworks.jira.com/servicedesk/customer/portal/12"
           sx={{
-            pt: 4,
-            mb: 2,
-            mx: 'auto',
-            width: 'max-content',
-            borderTop: '6px solid #ffffff88',
+            '*': { color: '#FFF', borderTop: 'none' },
```

Comment:
I suggest making the `color` `'primary.contrastText'`, which will make it more clear that you're setting the color to contrast with the background (which is the primary color). And I think it might make more sense to move this `color` style to the same `Box` where the `backgroundColor` is set to primary.main.

I am also not a big fan of using the `*` selector here either, but it's not obvious to me if there's a better way.

Reply:
Another change to consider is if the `borderTop` should be on the `h2` or the container surrounding `h2` (how you have it now).  If you had it on the `h2`, then you wouldn't need all these custom border styles here, and `PortalSectionHeader` component would just handle it.

In other cases, you would probably need to override `h2` to be `width: 100%` so the border takes up the full width of the container. I'm not 100% sure if this idea is an improvement, what do you think of it?